src/summary/level-two-routes.component.tsx

```diff
@@ -38,6 +39,16 @@ export function LevelTwoRoutes(props: LevelTwoRoutesProps) {
               />
             )}
           />
+          <Route
+            exact
+            path={`/patient/:patientUuid/chart/notes`}
+            render={routeProps => (
+              <PatientNotes
+                match={props.match}
```

Comment:
`props.match` is the coming from the LevelTwoRoutes props, not from the actual route for PatientNotes. This is a problem. Follow [this pattern from the react-router docs](https://reacttraining.com/react-router/web/api/Route/render-func) instead

```suggestion
                {...routeProps}
```